src/helper/site-utils.php

```diff
@@ -341,6 +347,38 @@ function create_etc_hosts_entry( $site_url ) {
 	}
 }
 
+/**
+ * Setup dnsmasq for darwin to resolve `*.test` domain.
+ *
+ * @return bool success.
+ */
+function setup_dnsmasq_for_darwin() {
+
+	// check if dnsmasq is installed.
+	if ( ! EE::exec( 'brew ls --versions dnsmasq' ) ) {
+		return false;
+	}
+
+	// create config directory.
+	EE::exec( 'mkdir -pv $(brew --prefix)/etc/' );
```

Comment:
```suggestion
	EE::exec( 'mkdir -p $(brew --prefix)/etc/' );
```

From the man pages, `-v` is not recommended